### ISSUE TITLE
commitAndReleaseBuffer: invalid argument (invalid character) error on windows ?

### ISSUE THREAD

**Issue by simonmichael** [joelburget/easytest#11]
My passing tests end with this error on Appveyor:
```
    OK Commands.show dollars
    OK Commands.show hours
    ------------------------------------------------------------
    test.exe: <stdout>: commitAndReleaseBuffer: invalid argument (invalid character)
```
I suspect the always-displayed unicode characters are a problem on Windows generally, and that they should be either disabled or have the output encoding set explicitly as discussed on https://github.com/hedgehogqa/haskell-hedgehog/issues/110 .

**Comment by joelburget** [comment]
I suspect this is fixed by #10. Though I'd also be fine with just removing the problematic characters. I'll cut a release with that fix and a fix for #12 as soon as it's fixed.

**Comment by simonmichael** [comment]
Yes, tests are running error-free on appveyor again. Thanks!
